pkg/workloadmeta/dump.go

```diff
@@ -33,6 +34,9 @@ func (wdr WorkloadDumpResponse) Write(writer io.Writer) {
 
 	for kind, entities := range wdr.Entities {
 		for entity, info := range entities.Infos {
+			info = scrubMessage(info)
```

Comment:
I don't think this solves the issue of sensitive data in a JSON blob from docker label. Or am I missing something ?